### PR TITLE
Feat/project/9mongodb

### PR DESCRIPTION
**Pull request by kamolsavar**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: todo_app/view_model.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def __init__(self, items):
     def todo_items(self):
         output = []
         for item in self._items:
-            if item.status == "ToDo":
+            if item.status == "To Do":
```
Comment (hugh-emerson) [inline todo_app/view_model.py:12]: Your update status button in index.html still sets status to ToDo

File: .github/workflows/todo.yml [hunk 2]
```diff
@@ -30,9 +29,8 @@ jobs:
     # if: contains('refs/heads/production', github.ref)  
     env:
       HEROKU_API_KEY: ${{secrets.HEROKU_API_KEY}}
-      TOKEN: ${{ secrets.trelloToken}}
-      KEY: ${{ secrets.trelloKey}}
-      SECRET_KEY: ${{ secrets.trelloSecretKey}}
+      MONGO_DB_CONNECTION: ${{ secrets.MONGO_DB_CONNECTION}}
+      DATABASE_NAME: ${{ secrets.DATABASE_NAME}}
```
Comment (hugh-emerson) [inline .github/workflows/todo.yml:33]: These env vars aren't actually used by this job. The Test job needs them to run the e2e tests, but you don't need these to deploy to Heroku.

You **could** use the Heroku CLI to set your app's env vars on Heroku, but you aren't currently. I assume you set them manually via the website, in which case you can delete these two lines.

**Review by hugh-emerson — APPROVED** [review]
Your README is now out of date, but otherwise everything looks good! I've suggested some small changes you could make but merge when you're ready 👍

File: todo_app/view_model.py (hunk 1 above)
Comment (kamolsavar) [reply todo_app/view_model.py:12]: Done

**Pull request merged**